dll/win32/shell32/utils.cpp

```diff
@@ -1294,3 +1294,30 @@ SHGetRealIDL(
 
     return hr;
 }
+
+static HRESULT
+GetCommandStringA(IContextMenu*pCM, UINT_PTR Id, UINT GCS, LPSTR Buf, UINT cchMax)
```

Comment:
Could we have SAL2 annotations for all the new code that we don't share with Wine? thanks.